homework_07/homework_07.py

```diff
@@ -0,0 +1,65 @@
+# homework_07
+
+phone_book = {}
+
+while True:
+    user_input = input('\n Select and enter a command \n'
+                       '(stats, add, delete, list, show): ')
+    print(user_input)
+
+    # stats: кількість записів
+    # add: додати запис
+    # delete <name>: видалити запис за іменем (ключем)
+    # list: список всіх імен в книзі
+    # show <name>: детальна інформація по імені
+
+    # user_input = user_input.split()
+    # print(user_input)
+
+    try:
+        if user_input == "add":
```

Comment:
The task says that there shouldn't be overriding of records. Please, fix it

Reply:
But this is not a redefinition, but a comparison. I don't know what needs to be changed. 
just need to add a check for the presence of the key of the same name before the "add" command?

Reply:
The comment is about the whole of this part. Try to create a record `test` and then try to create it again. Users shouldn't be able to override records, but in your case, they can do it